cmd/rsample/main.go

```diff
@@ -0,0 +1,209 @@
+// Shows Redis ratelimit groups statistics by checking given number of keys using RANDOMKEY.
+//
+// Example output
+// ===
+// ```
+// $ bin/rsample -swarm-redis-urls=localhost:6379 -samples=1000
+// shard                               keys
+// localhost:6379                      2468
+// total                               2468
+//
+// group                            samples    %total       min       max     50.0%     75.0%     95.0%     99.0%     99.9%
+// app-one-two-three                    381      38.1         1         3         2         3         3         3         3
+// app-one-two                          347      34.7         1         3         2         3         3         3         3
+// app-three                            149      14.9         1         1         1         1         1         1         1
+// app-two                              123      12.3         1         1         1         1         1         1         1
+// ```
+//
+// When sampling multiple Redis shards it is important to configure shard list in exactly the same order
```

Comment:
This is because of the ring, for the sampling the ring is not necessary so the improvement would be to ger rid of the ring and use a list of redis clients

Reply:
Needs verification as ring consistent hash should not depend on the member list order